### ISSUE TITLE
Cannot create Redshift table through sqlalchemy-redshift

### ISSUE THREAD

**Issue by larshelge** [apache/incubator-superset#9948]
## Description

I am trying to create a Redshift table in Apache Superset. I have first created the Redshift database, and verified that the connection is good. I have verified that the schema and the table I am creating actually exists in the Redshift cluster. I have installed sqlalchemy-redshift.

## How to reproduce the bug

- Create a Redshift database, and verify that the connection is succesful.
- Create a table based on the Redshift database, and ensure the schema and table exists in Redshift.
- Observe the error message message being displayed in the UI:

```
Table [public.dhis_dev] could not be found, please double check your database connection, schema, and table name, error: _get_column_info() missing 1 required positional argument: 'generated'
```

## Expected results

I want the creation of the Redshift table to be successful.

## Environment

sqlalchemy-redshift: 0.7.8
sqlalcemy: 1.3.17
superset: 0.36.0
pip: 20.1.1
python: 3.6.9
os: Ubuntu 18.04

## Error logs

After creating the table in Superset, I get the following error in the logs:

```
Got an error in pre_add for public.dhis_dev
Traceback (most recent call last):
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/superset/connectors/sqla/views.py", line 389, in pre_add
    table.get_sqla_table_object()
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/superset/connectors/sqla/models.py", line 1074, in get_sqla_table_object
    return self.database.get_table(self.table_name, schema=self.schema)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/superset/models/core.py", line 580, in get_table
    autoload_with=self.get_sqla_engine(),
  File "<string>", line 2, in __new__
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/util/deprecations.py", line 139, in warned
    return fn(*args, **kwargs)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 559, in __new__
    metadata._remove_table(name, schema)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 554, in __new__
    table._init(name, metadata, *args, **kw)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 648, in _init
    resolve_fks=resolve_fks,
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 672, in _autoload
    _extend_on=_extend_on,
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 2221, in run_callable
    return conn.run_callable(callable_, *args, **kwargs)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1660, in run_callable
    return callable_(self, *args, **kwargs)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 467, in reflecttable
    table, include_columns, exclude_columns, resolve_fks, **opts
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/reflection.py", line 666, in reflecttable
    table_name, schema, **table.dialect_kwargs
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/reflection.py", line 392, in get_columns
    self.bind, table_name, schema, info_cache=self.info_cache, **kw
  File "<string>", line 2, in get_columns
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/reflection.py", line 52, in cache
    ret = fn(self, con, *args, **kw)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy_redshift/dialect.py", line 461, in get_columns
    comment=col.comment)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy_redshift/dialect.py", line 664, in _get_column_info
    **kw
TypeError: _get_column_info() missing 1 required positional argument: 'generated'
ERROR:superset.connectors.sqla.views:Got an error in pre_add for public.dhis_dev
Traceback (most recent call last):
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/superset/connectors/sqla/views.py", line 389, in pre_add
    table.get_sqla_table_object()
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/superset/connectors/sqla/models.py", line 1074, in get_sqla_table_object
    return self.database.get_table(self.table_name, schema=self.schema)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/superset/models/core.py", line 580, in get_table
    autoload_with=self.get_sqla_engine(),
  File "<string>", line 2, in __new__
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/util/deprecations.py", line 139, in warned
    return fn(*args, **kwargs)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 559, in __new__
    metadata._remove_table(name, schema)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 554, in __new__
    table._init(name, metadata, *args, **kw)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 648, in _init
    resolve_fks=resolve_fks,
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 672, in _autoload
    _extend_on=_extend_on,
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 2221, in run_callable
    return conn.run_callable(callable_, *args, **kwargs)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1660, in run_callable
    return callable_(self, *args, **kwargs)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 467, in reflecttable
    table, include_columns, exclude_columns, resolve_fks, **opts
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/reflection.py", line 666, in reflecttable
    table_name, schema, **table.dialect_kwargs
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/reflection.py", line 392, in get_columns
    self.bind, table_name, schema, info_cache=self.info_cache, **kw
  File "<string>", line 2, in get_columns
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/reflection.py", line 52, in cache
    ret = fn(self, con, *args, **kw)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy_redshift/dialect.py", line 461, in get_columns
    comment=col.comment)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy_redshift/dialect.py", line 664, in _get_column_info
    **kw
TypeError: _get_column_info() missing 1 required positional argument: 'generated'
/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/flask_appbuilder/fields.py:182: UserWarning: allow_blank=True does not do anything for QuerySelectMultipleField.
  "allow_blank=True does not do anything for QuerySelectMultipleField."
```

### Checklist

Make sure these boxes are checked before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by willbarrett** [comment]
@dpgaspar this may be a FAB bug, not sure from the stacktrace.

**Comment by bryzgaloff** [comment]
I have encountered the same issue. It was fixed by installing [sqlalchemy-redshift==0.7.9](https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/releases/tag/0.7.9). My environment:
* apache-superset==0.37.2
* SQLAlchemy==1.3.19
* sqlalchemy-redshift==0.7.9
